modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/upgrade-catch-all-check/LPD_47113.testjava

```diff
@@ -0,0 +1,31 @@
+/**
+* SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+* SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+*/
+
+package com.liferay.source.formatter.dependencies.upgrade;
+
+import org.osgi.service.component.annotations.Reference;
+
+/**
+* @author Mateus Xavier
+*/
+public class LPD_47113 {
+
+    public void method(
+        long userId, long groupId, String name, int type, boolean auto,
+        ServiceContext serviceContext) {
+
+        SiteNavigationMenuLocalServiceUtil.addSiteNavigationMenu(123, groupId, name, type, auto, serviceContext);
+        SiteNavigationMenuServiceUtil.addSiteNavigationMenu(123, name, type, auto, serviceContext);
+        _siteNavigationMenuLocalService.addSiteNavigationMenu(userId, groupId, name, type, auto, serviceContext);
+        _siteNavigationMenuService.addSiteNavigationMenu(groupId, name, type, auto, serviceContext);
```

Comment:
@mxs2 only one message case its necessary, and all failing case should have a success one to test it if is it working.

And the only local services has the usedId parameter, you mixed the calls here.  Shouldn't be?

```
        SiteNavigationMenuLocalServiceUtil.addSiteNavigationMenu(123, groupId, name, type, auto, serviceContext);
       	SiteNavigationMenuLocalServiceUtil.addSiteNavigationMenu(userId, groupId, name, type, auto, serviceContext);
        SiteNavigationMenuServiceUtil.addSiteNavigationMenu(groupId, name, type, auto, serviceContext);
        _siteNavigationMenuLocalService.addSiteNavigationMenu(userId, groupId, name, type, auto, serviceContext);
        _siteNavigationMenuService.addSiteNavigationMenu(groupId, name, type, auto, serviceContext);
```